internal/drand-cli/proposal_file.go

```diff
@@ -68,6 +68,7 @@ type TomlParticipant struct {
 	Address   string
 	Key       string
 	Signature string
+	TLS       bool
 }
 
 func (t *TomlParticipant) Into() *drand.Participant {
```

Comment:
I think we need to add TLS here, no? And also below in the TOML() function and encode ones?